src/index.js

```diff
@@ -3,14 +3,16 @@ const glob = require('glob');
 const path = require('path');
 const util = require('util');
 const TapMochaReporter = require('tap-mocha-reporter');
+const c = require("ansi-colors");
```

Comment:
👶 nit: match the existing style convention for single vs double quotes?

Reply:
ah true, this file is split anyway so maybe i'll just convert them all